projects/igniteui-angular/src/lib/grids/toolbar/grid-toolbar-exporter.component.ts

```diff
@@ -134,16 +134,17 @@ export class IgxGridToolbarExporterComponent extends BaseToolbarDirective {
 
         this.exportStarted.emit(args);
         this.grid.onToolbarExporting.emit(args);
+        this.isExporting = true;
+        this.toolbar.showProgress = true;
```

Comment:
Okay, this is different from the old logic that waited at least 500ms to show the indicator but at least much simpler. I was worried it might flicker too easily, though same can be said for an export that lasts 505ms, so the old way didn't protect from that either. Seems to be okay as is, if you don't mind the unfinished progress animation:
![export-prog](https://user-images.githubusercontent.com/3198469/98656397-47da3680-2349-11eb-9d6c-fa881d61eca7.gif)
Perhaps we should consider minimum duration as an enhancement.